_data/versions.yml

```diff
@@ -1,4 +1,43 @@
 v2.1:
+- title: v2.1.3
+  note: |
+    * *Actually* Restore setting of `FELIX_DEFAULTENDPOINTTOHOSTACTION` for container environments,
+    which restores workloads' ability to reach services on its host.
+    * Bump calicoctl and calico/node to v1.1.1 they they pick up the new version of Felix
+  components:
+    felix:
+      version: 2.1.1
+      url: https://github.com/projectcalico/felix/releases/tag/2.1.1
+    calicoctl:
+      version: v1.1.1
+      url: https://github.com/projectcalico/calicoctl/releases/tag/v1.1.1
+      download_url: https://github.com/projectcalico/calicoctl/releases/download/v1.1.1/calicoctl
+    calico/node:
+      version: v1.1.1
+      url: https://github.com/projectcalico/calicoctl/releases/tag/v1.1.1
+    calico/cni:
+      version: v1.6.1
+      url: https://github.com/projectcalico/cni-plugin/releases/tag/v1.6.1
+      download_calico_url: https://github.com/projectcalico/cni-plugin/releases/download/v1.6.1/calico
+      download_calico_ipam_url: https://github.com/projectcalico/cni-plugin/releases/download/v1.6.1/calico-ipam
+    libcalico-go:
+      version: v1.1.1
+      url: https://github.com/projectcalico/libcalico-go/releases/tag/v1.1.0
```

Comment:
this needs to be v1.1.1 also I believe.  Though felix is actually using v1.1.4. https://github.com/projectcalico/felix/blob/2.1.1/glide.yaml#L23